sklearn/ensemble/tests/test_voting_classifier.py

```diff
@@ -428,3 +428,23 @@ def test_transform():
             eclf3.transform(X).swapaxes(0, 1).reshape((4, 6)),
             eclf2.transform(X)
     )
+
+    
+def test_voting_classifier_verbose():
+    
+    clf1 = LogisticRegression(random_state=123)
+    clf2 = RandomForestClassifier(random_state=123)
+    clf3 = GaussianNB()
+    X = np.array([[-1.1, -1.5], [-1.2, -1.4], [-3.4, -2.2], [1.1, 1.2]])
+    y = np.array([1, 1, 2, 2])
+
+    eclf1 = VotingClassifier(estimators=[
+        ('lr', clf1), ('rf', clf2), ('gnb', clf3)],
+        voting='soft',
+        n_jobs=1, verbose = 1)
```

Comment:
please also check that there is no verbosity unless verbose is set explicitly to a non-zero value.

Reply:
@jnothman , thanks for your help. But I am unable to test on my pc using the tests mentioned in docs. Is this ok?
def test_voting_classifier_verbose():
    
    clf1 = LogisticRegression(random_state=123)
    clf2 = RandomForestClassifier(random_state=123)
    clf3 = GaussianNB()
    X = np.array([[-1.1, -1.5], [-1.2, -1.4], [-3.4, -2.2], [1.1, 1.2]])
    y = np.array([1, 1, 2, 2])

    eclf1 = VotingClassifier(estimators=[
        ('lr', clf1), ('rf', clf2), ('gnb', clf3)],
        voting='soft',
        n_jobs=1, verbose = 1)
    old_stdout = sys.stdout
    sys.stdout = StringIO()
    try:
        eclfl.fit(X,y)
    finally:
        sys.stdout = old_stdout

Reply:
@jnothman out of curiosity, why does sklearn switch between using verbosity levels and boolean values? For example, in `sklearn/neural_network/multiplayer_perceptron.py`,

```
        if self.verbose is True or self.verbose >= 1:
            iprint = 1
        else:
            iprint = -1
```
a verbosity check for both number and boolean values are made.